platform/src/server/http/http_config.ts

```diff
@@ -18,8 +18,16 @@ const createHttpSchema = object({
   port: number({
     defaultValue: 5601,
   }),
+  cors: oneOf([
```

Comment:
__note:__ it's not exactly replicating schema that is defined by old Kibana since validate method in `@kbn/platform` can't get access to `env` config value right now. But wrong values will be rejected by Joi and won't reach `@kbn/platform` so it shouldn't be a big problem right now.

Reply:
We should probably create an issue or something to track adding this `env` support to the new platform, otherwise I could easily see us creating a regression here when we kill off the old platform. What do you think?

Reply:
Yeah, _if_ we really want to have functionality similar to Joi (references to _any_ config value during validation and default value initialization) we can build it, although it won't be a trivial task since we'll have to parse and validate config in the right order.

Reply:
Will be handled separately based on https://github.com/elastic/kibana/issues/18818